usaspending_api/awards/serializers_v2/serializers.py

```diff
@@ -15,3 +19,412 @@ class RecipientAwardSpendingSerializer(serializers.Serializer):
+        pattern = "{}__".format(child)
+
+        matched = []
+        child_fields = []
+        for field in fields:
+            if field[:len(pattern)] == pattern:
+                child_fields.append(field[len(pattern):])
+                matched.append(field)
+        return child_fields, matched
+
+
+class LegalEntityOfficersSerializerV2(LimitableSerializerV2):
+    class Meta:
+        model = LegalEntityOfficers
+        fields = [
+            "officer_1_name",
+            "officer_1_amount",
+            "officer_2_name",
+            "officer_2_amount",
+            "officer_3_name",
+            "officer_3_amount",
+            "officer_4_name",
+            "officer_4_amount",
+            "officer_5_name",
+            "officer_5_amount",
+        ]
+
+
+class ToptierAgencySerializerV2(LimitableSerializerV2):
+
+    class Meta:
+        model = ToptierAgency
+        fields = [
+            "name",
+            "abbreviation"
+        ]
+
+
+class SubtierAgencySerializerV2(LimitableSerializerV2):
+
+    class Meta:
+        model = SubtierAgency
+        fields = [
+
+            "name",
+            "abbreviation"
+        ]
+
+
+class OfficeAgencySerializerV2(LimitableSerializerV2):
+
+    class Meta:
+        model = OfficeAgency
+        fields = [
+            "aac_code",
+            "name"
+        ]
+
+
+class LocationSerializerV2(LimitableSerializerV2):
+
+    class Meta:
+        model = Location
+        fields = [
+            "address_line1",
+            "address_line2",
+            "address_line3",
+            "foreign_province",
+            "city_name",
+            "county_name",
+            "state_code",
+            "zip5",
+            "zip4",
+            "foreign_postal_code",
+            "country_name",
+            "location_country_code",
+            "congressional_code"
+        ]
+
+
+class AgencySerializerV2(LimitableSerializerV2):
+    office_agency_name = serializers.SerializerMethodField('office_agency_name_func')
+
+    def office_agency_name_func(self, agency):
+        if agency.office_agency:
+            return agency.office_agency.name
+        else:
+            return None
+
+    class Meta:
+        model = Agency
+        fields = [
+            "toptier_agency",
+            "subtier_agency",
+            "office_agency_name",
+        ]
+        nested_serializers = {
+            "toptier_agency": {
+                "class": ToptierAgencySerializerV2,
+                "kwargs": {"read_only": True}
+            },
+            "subtier_agency": {
+                "class": SubtierAgencySerializerV2,
+                "kwargs": {"read_only": True}
+            },
+            "office_agency": {
+                "class": OfficeAgencySerializerV2,
+                "kwargs": {"read_only": True}
+            },
+        }
+
+
+class LegalEntitySerializerV2(LimitableSerializerV2):
+    '''recipient_parent_name = serializers.SerializerMethodField('recipient_parent_name_func')
```

Comment:
I don't see a benefit leaving this in the codebase since it won't be used in the future.